scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -139,189 +139,7922 @@
+	language	Estonian Sign Language	eso	1878	eso
+	language	Esuma	esm	1879	esm
+	language	Etchemin	etc	1880	etc
+	language	Etebi	etb	1881	etb
+	language	Eten	etx	1882	etx
+	language	Eteocretan	ecr	1883	ecr
+	language	Eteocypriot	ecy	1884	ecy
+	language	Ethiopian Sign Language	eth	1885	eth
+	language	Etkywan	ich	1886	ich
+	language	Eton (Cameroon)	eto	1887	eto
+	language	Eton (Vanuatu)	etn	1888	etn
+	language	Etruscan	ett	1889	ett
+	language	Etulo	utr	1890	utr
+	language	Eudeve	eud	1891	eud
+	language	Evant	bzz	1892	bzz
+	language	Even	eve	1893	eve
+	language	Evenki	evn	1894	evn
+	language	Eviya	gev	1895	gev
+	language	Ewage-Notu	nou	1896	nou
+	language	Ewe	ewe	1897	ewe	ee
+	language	Ewondo	ewo	1898	ewo
+	language	Extremaduran	ext	1899	ext
+	language	Eyak	eya	1900	eya
+	language	Ezaa	eza	1901	eza
+	language	Fa d'Ambu	fab	1902	fab
+	language	Fagani	faf	1903	faf
+	language	Faifi	fif	1904	fif
+	language	Faire Atta	azt	1905	azt
+	language	Faita	faj	1906	faj
+	language	Faiwol	fai	1907	fai
+	language	Fala	fax	1908	fax
+	language	Falam Chin	cfm	1909	cfm
+	language	Fali	fli	1910	fli
+	language	Faliscan	xfa	1911	xfa
+	language	Fam	fam	1912	fam
+	language	Fanagalo	fng	1913	fng
+	language	Fanamaket	bjp	1914	bjp
+	language	Fanbak	fnb	1915	fnb
+	language	Fang (Cameroon)	fak	1916	fak
+	language	Fang (Equatorial Guinea)	fan	1917	fan
+	language	Fania	fni	1918	fni
+	language	Fanti	fat	1919	fat
+	language	Far Western Muria	fmu	1920	fmu
+	language	Farefare	gur	1921	gur
+	language	Faroese	fao	1922	fao	fo
+	language	Fas	fqs	1923	fqs
+	language	Fasu	faa	1924	faa
+	language	Fataleka	far	1925	far
+	language	Fataluku	ddg	1926	ddg
+	language	Fayu	fau	1927	fau
+	language	Fe'fe'	fmp	1928	fmp
+	language	Fembe	agl	1929	agl
+	language	Fernando Po Creole English	fpe	1930	fpe
+	language	Feroge	fer	1931	fer
+	language	Fiji Hindi	hif	1932	hif
+	language	Fijian	fij	1933	fij	fj
+	language	Filipino	fil	1934	fil
+	language	Filomena Mata-Coahuitlán Totonac	tlp	1935	tlp
+	language	Finland-Swedish Sign Language	fss	1936	fss
+	language	Finnish	fin	1937	fin	fi
+	language	Finnish Sign Language	fse	1938	fse
+	language	Finongan	fag	1939	fag
+	language	Fipa	fip	1940	fip
+	language	Firan	fir	1941	fir
+	language	Fiwaga	fiw	1942	fiw
+	language	Flaaitaal	fly	1943	fly
+	language	Flinders Island	fln	1944	fln
+	language	Foau	flh	1945	flh
+	language	Foi	foi	1946	foi
+	language	Foia Foia	ffi	1947	ffi
+	language	Folopa	ppo	1948	ppo
+	language	Foma	fom	1949	fom
+	language	Fon	fon	1950	fon
+	language	Fongoro	fgr	1951	fgr
+	language	Foodo	fod	1952	fod
+	language	Forak	frq	1953	frq
+	language	Fordata	frd	1954	frd
+	language	Fore	for	1955	for
+	language	Forest Enets	enf	1956	enf
+	language	Fortsenal	frt	1957	frt
+	language	Francisco León Zoque	zos	1958	zos
+	language	Frankish	frk	1959	frk
+	language	French	fra	1960	fra	fre	fr
+	language	French Sign Language	fsl	1961	fsl
+	language	Friulian	fur	1962	fur
+	language	Fula, Fulah, Pular, Pulaar	ful	1963	ful	ff	Fula	Fulah	fuc	Pular	fuf	Pulaar
```

Comment:
In iso-639-3.tab, I see
```
fuc				I	L	Pulaar	
fuf				I	L	Pular	
ful	ful	ful	ff	M	L	Fulah	

```
which means these are not the same language (as I assume they were considered on 639-1 or 639-2 where we got the original entry with all of these together.)

It looks like 639-3 considers ful to be the macro language comprised of ffm, fub, fuc, fue, fuf, fuh, fui, fuq, and fuv (from https://iso639-3.sil.org/sites/iso639-3/files/downloads/iso-639-3-macrolanguages.tab). We have all but fuc and fuf as separate entries now.

I think fuc/fuf should also be separate and, in general, the algorithm doing the merge should untangle an alternates that are considered a separate language in 639-1. It looks like there are ~64 macrolanguages and probably few of them are represented in our old table. For those that are, I think any alternates that are for separate individual languages should keep their separate entries.

Reply:
If I remove Pulaar and Pular from our original tsv file the merge will make them separate. I'll do that